zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -140,6 +140,35 @@ <h1>Edit upload</h1>
       <h1>New version</h1>
       {%- else %}
       <h1>New upload</h1>
+      {% if record.files %}
+      {%- if current_user_is_admin() %}
+      <div class="btn-group btn-block">
+        <button type="button" class="btn btn-m btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Admin <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu">
+          <li><a href="{{ url_for('persistentidentifier.index_view', flt1_21=record.id) }}">
+            Persistent identifiers
+          </a></li>
+          <li><a href="{{ url_for('recordmetadata.details_view', id=record.id) }}">
+            Record
+          </a></li>
+          <li><a href="{{ url_for('bucket.edit_view', id=record.files.bucket) }}">
```

Comment:
```suggestion
          <li><a href="{{ url_for('bucket.edit_view', id=record.files.bucket.id) }}">
```

It kind of works by accident I feel now (since if you do `print(bucket)` you get [`bucket.id`](https://github.com/inveniosoftware/invenio-files-rest/blob/27e067be62c0b76e96428397cae84555b016f614/invenio_files_rest/models.py#L388))